component/lists/list.css

```diff
@@ -0,0 +1,58 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: var(--primary-font);

```

Comment:
It's not against the rules in using * selector which means any/all elements, but will obviously be on the bottom of the property chain when it comes to overriding more specific selectors. I guess the best practice is to use inside body.